docs/gui/menus.md

```diff
@@ -319,8 +326,8 @@ public class MyMob extends Mob implements MenuProvider {
 
   @Override
   public InteractionResult mobInteract(Player player, InteractionHand hand) {
-    if (!this.level.isClientSide && player instanceof ServerPlayer serverPlayer) {
-      NetworkHooks.openScreen(serverPlayer, this);
+    if (!this.level.isClientSide && player instanceof IForgeServerPlayer serverPlayer) {
+      serverPlayer.openMenu(this, new BlockPos(this.getX(), this.getY(),this.getZ()));
```

Comment:
`Entity#blockPosition` exists instead of creating it yourself.

Reply:
I veiw the NetworkHook and find out that it is not need for us to provide a pos to entityMenu.If the old doc is correct.
public static void openScreen(ServerPlayer player, MenuProvider containerSupplier)
    {
        openScreen(player, containerSupplier, buf -> {});
    }